api/stats.js

```diff
@@ -45,20 +87,10 @@ export class Stats {
             }
         }
 
-
         return stats;
     }
 
-    async postDailyStats() {
-        let dominationGuild = this.client.guilds.resolve(config.GUILD_ID);
-        let channels = await dominationGuild.channels.fetch();
-        const today = new Date();
-        today.setHours(0, 0, 0, 0);
-        // https://discord.com/developers/docs/reference#snowflakes
-        let midnight = SnowflakeUtil.generate(today);
-        let stats = await this._buildStats(channels.values(), midnight);
-        console.log(stats);
-
+    async _sendStatsChannel(stats, title, channel) {
```

Comment:
I reckon an early validation here if 'stats' has any content would be beneficial. Is probably not the case for `stats` to be ever empty, however just in case.

Reply:
added the validation in send stats